### ISSUE TITLE
Slightly malformed markdown due to Contentful editor

### ISSUE THREAD

**Issue by FickleLife** [ModiiMedia/contentful-hugo#27]
When an editor is modifying content in Contentful, they have to be super careful not ensure that white spaces are not bolded/itialcised otherwise the markdown when displayed in hugo is not translated correctly. For example, in Contentful the text looks like this: 

- **_My Score_** - the ability for you to customise how the score is calculated to your own personalised score, even to exclude products (show in red) that do not meet your criteria.

after run through contentful-hugo, this is what has been generated in markdown: 
```
- ***My Score ***- the ability for you to customise how the score is calculated to your own personalised score, even to exclude products (show in red) that do not meet your criteria.

```

and this is what is on screen with Hugo: https://imgur.com/Zeo4pIX

Here's the HTML generated: 
```
<li>***My Score ***- the ability for you to customise how the score is calculated to your own personalised score, even to exclude products (show in red) that do not meet your criteria.</li>
```

And taking a different paragraph that has it's formatting perfect (no bold or italics on spaces) in Contentful, the output looks like:

```
<li><strong>This is another test paragraph</strong> which comes in two flavours:</li>
```

Note the space after `My Score`, this seems to be what causes issues in output as it appears that the `***` doesn't end at the end of `Score` therefore it is deemed to start at `-`.

Looking at the raw content from the contentful API, the data looks like: 

```
{
                                                    "nodeType": "list-item",
                                                    "content": [
                                                        {
                                                            "nodeType": "paragraph",
                                                            "content": [
                                                                {
                                                                    "nodeType": "text",
                                                                    "value": "My Score ",
                                                                    "marks": [
                                                                        {
                                                                            "type": "bold"
                                                                        },
                                                                        {
                                                                            "type": "italic"
                                                                        }
                                                                    ],
                                                                    "data": {}
                                                                },
                                                                {
                                                                    "nodeType": "text",
                                                                    "value": "- the ability for you to customise how the score is calculated to your own personalised score, even to exclude products (show in red) that do not meet your criteria.",
                                                                    "marks": [],
                                                                    "data": {}
                                                                }
                                                            ],
                                                            "data": {}
                                                        }
                                                    ],
                                                    "data": {}
                                                }
```

The space after `My Score` seems to be causing issues things up.... So I guess I have two questions

1. Can/should contentful-hugo "repair" these formatting issues when generating? I imagine this is a rabbit hole....
2. Is the answer to just ask contentful editors to be extremely careful when formatting the content to ensure there's no bold/italics set on spaces? 

I'm dealing with end users who are not very meticulous (and are finding it difficult to change)when it comes to 2 as it all looks perfect in Contentful but renders out incorrectly in Hugo, so am sounding this out here to see what the consensus is....

**Comment by joshmossas** [comment]
Hey @FickleLife apologies as I'm just now seeing this.

I think we could actually repair this particular issue for marks, such as bold and italic text. I don't think it's reasonable to expect the end users to have to be that meticulous when writing content.

**Comment by joshmossas** [comment]
@FickleLife this issue is now fixed in 1.9.2. 

I've run tests with italic marks, bold marks, and combinations of the two. Please let me know if you are still having problems and I'll reopen the issue 😄
